Tests/LibAudio/TestWav.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2024, Lee Hanken <github-12-2017-ds8@leehanken.uk>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ByteString.h>
+#include <AK/LexicalPath.h>
+#include <AK/ScopeGuard.h>
+#include <LibAudio/WavLoader.h>
+#include <LibAudio/WavWriter.h>
+#include <LibCore/Directory.h>
+#include <LibCore/File.h>
+#include <LibFileSystem/FileSystem.h>
+#include <LibFileSystem/TempFile.h>
+#include <LibTest/TestCase.h>
+
+static void compare_files(ByteString const in_path, ByteString const out_path)
```

Comment:
This method will fail if the metadata is different, we should probably compare samples instead. But I guess that's fine for now.